src/main.js

```diff
@@ -0,0 +1,451 @@
+"use strict";
+/**
+ * Количество карточек задачи на странице
+ * @type {number}
+ */
+const TASK_CARDS_COUNT = 3;
+
+
+// ГЛАВНЫЕ КОНТЕЙНЕРЫ:
+/**
+ * Главный контейнер страницы
+ * @type {Element}
+ */
+const mainContainerElement = document.querySelector(`.main`);
+
+/**
+ * Контейнер с меню
+ * @type {Element}
+ */
+const menuContainerElement = mainContainerElement.querySelector(`.control`);
```

Comment:
1. Это всё вкусовщина, но 11 строк вместо 2 скорее мешают чтению.
2. Комментарии не должны дублировать очевидную информацию из имени переменной.
3. Язык с типизацией очевидные вещи вроде `@type {Element}` выведет сам, поэтому я не вижу смысла руками всё дублировать, пусть даже и с помощью IDE.
 
```suggestion
const mainContainerElement = document.querySelector(`.main`);
const menuContainerElement = mainContainerElement.querySelector(`.control`);
```